### DISCUSSION TITLE
YAZIO App

### DISCUSSION THREAD

**Discussion by Cotrox** [zepp-health/zeppos-forum#358] [Q&A] [answered]
I'm thinking about a way to create a Yazio connected App. Do you think can be possible? Someone want to help me with it? I'm not skilled.

**Comment by Cotrox** [comment]
Are you here guys?

**Reply by maxiges** [reply, marked as answer]
It is absolutely possible to do this, because the entire application relies on the API, so you can do it. You just need to have the will and determination. See the examples on how to use the featch API.

**Reply by maxiges** [reply]
https://github.com/zepp-health/zeppos-samples/tree/main/application/3.0/fetch-api

https://github.com/saganos/yazio_public_api

**Reply by Cotrox** [reply]
Yeah but I don't know why but I'm having problems with the dev ambient. I used ZeppOS extebsion for Visual Studio Code but now it doesn't work.

**Reply by Cotrox** [reply]
Thank you! Do u wanna work on it with me? I'm not so skilled at it!

**Reply by Cotrox** [reply]
I solved zeus problems but i'm not sure about how to realize the code.
